workshops/forms.py

```diff
@@ -490,6 +477,17 @@ class Meta:
                   'email', 'gender', 'airport', 'affiliation', 'github',
                   'twitter', 'url', 'occupation', 'orcid', 'notes', 'lessons',
                   'domains', 'languages']
+        widgets = {
+            'airport': selectable.AutoComboboxSelectWidget(
+                lookup_class=lookups.AirportLookup,
+            ),
+            'languages': selectable.AutoComboboxSelectMultipleWidget(
+                lookup_class=lookups.LanguageLookup,
+            ),
+            'github': selectable.AutoCompleteSelectWidget(
```

Comment:
Is there any reason for using `AutoCompleteSelectWidget` instead of `AutoComboboxSelectWidget`?